docs/commands/ft.search.md

```diff
@@ -68,8 +68,9 @@ Searches the index with a textual query, returning either documents or just ids.
 
 - **SORTBY {attribute} [ASC|DESC]**: If specified, the results are ordered by the value of this attribute. This applies to both text and numeric attributes. Attributes needed for **SORTBY** should be declared as **SORTABLE** in the index, in order to be available with very low latency (notice this adds memory overhead)
 
-- **LIMIT first num**: Limit the results to
-  the offset and number of results given. Note that the offset is zero-indexed. The default is 0 10, which returns 10 items starting from the first result.
+- **LIMIT first num**: Limit the results to the offset and number of results given.
+  Note that the offset is zero-indexed. The default is 0 10, which returns 10 items starting from the first result.
+  If a result key expires during the query, its content will be a null-array.
```

Comment:
Should also be aligned with the "Expiration of hashes during a search query" below?
It says

> name and content of the hash will not be returned.

Should it be something like:
name and content of the key will return as null